pkg/rego/load.go

```diff
@@ -156,7 +156,7 @@ func (s *Scanner) LoadPolicies(loadEmbedded bool, srcFS fs.FS, paths []string, r
 	for namespace := range uniq {
 		namespaces = append(namespaces, namespace)
 	}
-	store, err := initStore(s.dataDirs, namespaces)
+	store, err := initStore(s.dataFS, s.dataDirs, namespaces)
```

Comment:
maybe `dataDirs` can be removed

Reply:
After removing `dataDirs` locally, I changed my mind. `dataFS` possibly contains irrelevant files and dirs. We might want to filter out those files/dirs. I keep `dataDirs` for the reason.